README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
"operator of x" block gives error:
> OPERATOR not found

Happens for all operators, not just "abs"
![image](https://user-images.githubusercontent.com/126523/220715891-b66d3e56-7b75-46c9-a590-64d936f8de8e.png)